content/Video_Display.md

```diff
@@ -556,6 +556,8 @@ to select the origin of the visible 160x144 pixel area within the total
 256x256 pixel background map. Background wraps around the screen (i.e.
 when part of it goes off the screen, it appears on the opposite side.)
 
+Background can be disabled(0)/enabled(1) with 0th bit of LCDC register.
```

Comment:
I don't think "with 0th" sounds good. Maybe *with the bit 0*?

Reply:
I'll look for better alternatives, maybe *LCDC.bit0* would be better.

Reply:
Just like above, the grammatically correct form would be "The background can be disabled or enabled with bit 0 of the LCDC register."